### ISSUE TITLE
Incomplete PtcimCollection.add()?

### ISSUE THREAD

**Issue by kolovos** [epsilonlabs/emc-artisan#23]
In its current form, PtcimCollection.add() doesn't seem to have any effect on the underlying model (it only constructs a temporary ArrayList and adds two objects to it).

```java
@Override
public boolean add(PtcimObject e) {
	assert e.getId() != null;
	List<Object> args = new ArrayList<Object>();
	args.add(association);
	args.add(e);
	return true;
}
```

**Comment by zolotas4** [comment]
Remove doesn't work properly as well...

**Comment by zolotas4** [comment]
Fixed in 0cfa2b13b57b00d23397c64002318ee07993150d
